cvat-canvas3d/src/typescript/canvas3dView.ts

```diff
@@ -0,0 +1,213 @@
+// Copyright (C) 2021 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import { Canvas3dController } from './canvas3dController';
+import { Listener, Master } from './master';
+
+import consts from './consts';
+import * as THREE from 'three';
```

Comment:
`three` import should occur before import of `./canvas3dController`eslintimport/order

Reply:
Resolved